msal-dotnet-articles/acquiring-tokens/web-apps-apis/on-behalf-of-flow.md

```diff
@@ -7,16 +7,16 @@ description: "How to use MSAL.NET to authenticate on behalf of a user."
 
 ## If you are using ASP.NET Core
 
-If you are building a web API on to of ASP.NET Core, we recommend that you use Microsoft.Identity.Web. See [Web APIs with Microsoft.Identity.Web](https://github.com/AzureAD/microsoft-identity-web/wiki/web-apis).
+If you are building a Web API on top of ASP.NET Core or ASP.NET Classic, we recommend that you use Microsoft.Identity.Web. See [Web APIs with Microsoft.Identity.Web](https://github.com/AzureAD/microsoft-identity-web/wiki/web-apis).
 
-You might want to check the decision tree: [Is MSAL.NET right for me?](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Is-MSAL.NET-right-for-me%3F).
+Check the decision tree: [Is MSAL.NET right for me?](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Is-MSAL.NET-right-for-me%3F).
```

Comment:
Let's link to docs instead of the wiki

```suggestion
Check the decision tree: [Is MSAL.NET right for me?](/entra/msal/dotnet/getting-started/choosing-msal-dotnet)
```